admin_guide/out_of_resource_handling.adoc

```diff
@@ -168,6 +168,27 @@ To configure hard eviction thresholds, the following flag is supported:
 * `eviction-hard`: a set of eviction thresholds (for example,
 `memory.available<1Gi`) that, if met, triggers a pod eviction.
 
+[[soft-and-hard-eviction-example-configuration]]
+==== Soft and Hard Eviction Example Configuration
+
+.Soft and Hard Eviction Example Configuration on a 32Gi Node
+----
+eviction-hard:
+- "memory.available<100Mi"
+eviction-soft:
+- "memory.available<1Gi"
+eviction-soft-grace-period:
+- "memory.available=60s"
```

Comment:
We probably don't want to be recommending changes to this setting. 
We likely want to be recommending that the default here is used (and explaining why).